contracts/lib/ens/AbstractENS.sol

```diff
@@ -1,14 +1,14 @@
-pragma solidity ^0.4.15;
```

Comment:
Contracts in the `lib` directory are just copied from their source because they went through code review already. Please revert changes in lib to their original version!